collectd/files/collectd_logrotate.conf

```diff
@@ -0,0 +1,11 @@
+/var/log/{{ service_name }}.log {
+  rotate 5
+  daily
+  nocreate
+  missingok
+  compress
+  delaycompress
+  minsize 20M
+  maxsize 50M
+  su root syslog
```

Comment:
I think that "su root syslog" is herited from the main logrotate configuration file, then is not mandatory here